test/features/helpers/i18n_helper.rb

```diff
@@ -0,0 +1,43 @@
+require 'fileutils'
+require 'yaml'
+require_relative '../utils/index.rb'
+
+LOCALES = ['en', 'es'] # List of supported locales
+
+class I18nHelper
+  def initialize(driver)
+    @driver = driver
+  end
+
+  def load_locale(locale)
+    tag = locale ? locale : 'en'
+    path_to_file = File.join(Dir.pwd, 'locales', "#{tag}.yml")
+    @translations = YAML.load_file(path_to_file)
+  end
+
+  def translate(key)
+    @translations[key]
+  end
+
+  def set_locales
+    LOCALES.each do | locale |
```

Comment:
I'm struggling to understand what is happening here and why. Could we discuss?

Reply:
Sure, I can talk you through it.